src/eval/mod.rs

```diff
@@ -177,6 +178,10 @@ impl<R: ImportResolver> VirtualMachine<R> {
         &mut self.import_resolver
     }
 
+    pub fn stack_mut(&mut self) -> &mut Stack {
```

Comment:
Wouldn't that be easier to implement stack unwiding directly in the `Stack` or `VirtualMachine`? We wouldn't have to expose the internals of `VirtualMachine` publicly.

Reply:
I'll look into it